app/assets/stylesheets/_variables.scss

```diff
@@ -26,29 +21,14 @@ $dark-gray: #333;
 $medium-gray: #999;
 $light-gray: #DDD;
 
-// Background Color
-$base-background-color: white;
-
 // Font Colors
 $base-font-color: $dark-gray;
-$base-accent-color: $blue;
-
-// Link Colors
-$base-link-color: $base-accent-color;
-$hover-link-color: darken($base-accent-color, 15);
-$base-button-color: $base-link-color;
-$hover-button-color: $hover-link-color;
+$action-color: $blue;
```

Comment:
I've started experimenting with global variables in Refills, taking as much as possible from Bitters. These are the names I've played around with instead of the more specific ones used here:

```
// Colors
$dark-color: #333;
$light-color: #ddd;
```

I've also found myself using both `base-font-color` and something such as `base-font-color-inverted` quite often, for dark areas (if `base-font-color` is dark, that is). Therefore I'm experimenting with the following for Refills:

// Font Colors
$base-font-color-dark: $dark-color;
$base-font-color-light: #fff;
$action-color-dark: #477dca;
$action-color-light: lighten($action-color-dark, 25%);

Ideally the variables for Refills and Bitters would be the same but they don't have to be. IF Refills gets variables...